tools/annotation_tools/turk_with_s3/run_tool_1A.py

```diff
@@ -34,14 +34,14 @@
 
 time.sleep(100)
```

Comment:
sorry if I'm being ignorant here: are we expecting the created jobs to be completed within 100 seconds?

Reply:
No, this is something to change for production. Though it's true that tasks are generally picked up really quickly.